### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by xotonic**

### CODE REVIEW

File: docs/introduction/di_in_python.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ using :doc:`Dependency Injector <../index>`:
    :language: python
    :linenos:
 
-Userful links
+Useful links
 ~~~~~~~~~~~~~
```
Comment (rmogylatov-cisco) [inline docs/introduction/di_in_python.rst]: Can you, please, also align this line with previous one? Number of `~` should match the length of the header.

**Comment by rmogylatov-cisco** [comment]
Hi @xotonic ,

Thanks a lot for the fix.

Can I, please, ask you to take a look on little comment above and also change base branch to "develop"?

Thanks,
Roman

**Comment by rmk135** [comment]
Awesome! Thank you!

**Comment by rmk135** [comment]
@xotonic ,

Release 3.14.7 with a fix has been productized:

- https://pypi.org/project/dependency-injector/3.14.7/
- http://python-dependency-injector.ets-labs.org/main/changelog.html#id1

I've added you to the list of contributors - https://github.com/ets-labs/python-dependency-injector/blob/master/CONTRIBUTORS.rst

Thanks a lot for your contribution - it's very appreciated!

Regards,
Roman

**Pull request merged**